src/python/pants/help/help_info_extracter.py

```diff
@@ -29,19 +29,33 @@ class OptionHelpInfo(namedtuple('_OptionHelpInfo',
   removal_hint: If deprecated: The removal hint message registered for this option.
   choices: If this option has a constrained list of choices, a csv list of the choices.
   """
+  registering_class: Type
+  display_args: List[str]
+  scoped_cmd_line_args: List[str]
+  unscoped_cmd_line_args: List[str]
+  typ: Type
+  default: str
+  help: str
+  deprecated_message: str
+  removal_version: str
+  removal_hint: str
```

Comment:
I think these should all be `Optional[str]`.

Reply:
Got it - `deprecated_message`, `removal_version`, `removal_hint`, and `choices` are now `Optional[str]`